README.md

```diff
@@ -65,7 +65,9 @@ The extension provides currently two high level `calculator` classes that wrap t
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+Before installing dependencies, make sure your environment variables are setup. Our CI uses something like this: `SOLIDUS_BRANCH=v2.10 DB=postgresql RAILS_VERSION="~> 5.2.0`. These environment variables need to be set when installing dependencies and running specs, so set them before the command like this: `SOLIDUS_BRANCH=v2.10 DB=postgresql RAILS_VERSION="~> 5.2.0 bin/setup`
```

Comment:
These variables all have default values so there shouldn't be any need to define them unless you're looking to target a specific setup. (So the "need to be set" is a little misleading.)

Reply:
This change also probably shouldn't be bundled in with the pull-request template PR.

Reply:
Ahh, I messed up again and didn't rebase correctly onto master.

>These variables all have default values so there shouldn't be any need to define them unless you're looking to target a specific setup. (So the "need to be set" is a little misleading.)

You actually do need to set them! Whatever the default values are don't work, if you just run `bin/setup` and `bin/rake` without any env. But maybe instead of adding this readme, we should figure out why the default values don't work.